docs/build-blockchain/products/v2/ui-interactions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: UI Interactions
+id: ui-interactions
+---
+
+# Premium account interactions
+
+To start using the _Premium Tier_ services, there are some required steps for you to go through:
+
+`Connect wallet` —>`Top up account balance` —>`Start using custom RPC endpoints` —>`Add email`
```

Comment:
suggesting spaces before and after ->

Reply:
Yeah, that could have been possible if not our CSS, lol. Spaces after create some shitty look.